src/wav.c

```diff
@@ -79,7 +79,7 @@
 
 #define WAVLIKE_PEAK_CHUNK_SIZE(ch) 	(2 * sizeof (int) + ch * (sizeof (float) + sizeof (int)))
 
-
+#define MIN(a,b) ((a) < (b) ? (a) : (b))
```

Comment:
Please use the already defined `SF_MIN()` macro instead.

Reply:
Ah, was searching for an existing macro, but for some reason it didn't come up. Thanks, will fix